pkgs/applications/networking/feedreaders/NetNewsWire/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  pname = "NetNewsWire";
+  version = "6.1.4";
+
+  src = fetchzip {
+    url =
+      "https://github.com/Ranchero-Software/${pname}/releases/download/mac-${version}/${pname}${version}.zip";
+    hash = "sha256-9RuvMklwFBi7KbR0P2HU1Mpe5t78W7CwtoxAkU5E2og=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/Applications
+    cp -R NetNewsWire.app $out/Applications/
+  '';
+
+  meta = with lib; {
+    description = "RSS reader for macOS and iOS.";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

```suggestion
    description = "RSS reader for macOS and iOS";
```